Bottom: 75f54c7c6044b08520a5aadf6833cd99d197cbbb
Top:    95958818378461c79150fce6a540a9a6556e9f24
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 23:27:50 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index e5fca5a..d1846eb 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
